dune-project

```diff
@@ -356,10 +363,12 @@ should also be fine under Windows too.
   "A CoHTTP server and client implementation based on `eio` library. `cohttp-eio`features a multicore capable HTTP 1.1 server. The library promotes and is built with direct style of coding as opposed to a monadic.")
  (depends
   base-domains
+  (cohttp
+   (= :version))
   (eio
    (and
-    (>= 0.10)
-    (< 0.12)))
+    (>= 0.12)
+    (< 0.13)))
```

Comment:
Shouldn't need an upper-bound here. We can add one later if it turns out to be needed, but we want to know if cohttp-eio isn't compatible with a new version of Eio.

Reply:
My rationale here is that the most important is that users build do not break out of the blue one morning (including their CI etc.) just because of a new eio release. One can use subminor versions for non-API breaking changes. I do however realize that there's no consensus on that.